docs/source/interfaces/ISIS SANS v2.rst

```diff
@@ -122,6 +122,128 @@ Save Options
+|       |                           | table.                                                                                  |
++-------+---------------------------+-----------------------------------------------------------------------------------------+
+| **4** | **Manage Directories**    | Opens the 'Manage User Directories' window allowing the user to add/remove directories  |
+|       |                           | from the mantid search path and to set the 'Output Folder' where the summation result   |
+|       |                           | will be saved.                                                                          |
++-------+---------------------------+-----------------------------------------------------------------------------------------+
+| **5** | **Remove**                | Removes an entry from the summation table. Note, this does not delete the file itself,  |
+|       |                           | it just removes it from the list of files to be summed.                                 |
++-------+---------------------------+-----------------------------------------------------------------------------------------+
+| **6** | **Remove All**            | Removes all entries from the summation table. As above, this will only remove the       |
+|       |                           | entries from the table, not the files themselves.                                       |
++-------+---------------------------+-----------------------------------------------------------------------------------------+
+
+Histogram Binning
+^^^^^^^^^^^^^^^^^
+
+This panel allows the user to specify the binning parameters to be applied when summing event data.
+There are three different ways to add files containing event data [#no-effect-when-non-event]_.
+
+Custom Binning
+""""""""""""""
+
+If this option is chosen a line edit field becomes available which the user can use to set the
+preferred binning boundaries. The format of this input is identical to the format required by the
+:doc:`Rebin Algorithm <../algorithms/Rebin-v1>`.
+
+    A comma separated list of first bin boundary, width, last bin boundary. Optionally this can be
+    followed by a comma and more widths and last boundary pairs. Optionally this can also be a single
+    number, which is the bin width. In this case, the boundary of binning will be determined by minimum
+    and maximum TOF values among all events, or previous binning boundary, in case of event Workspace,
+    or non-event Workspace, respectively. Negative width values indicate logarithmic binning.
+
+Use Binning From Monitors
+"""""""""""""""""""""""""
+
+If this option is chosen the binning is taken from the monitors.
+
+Save As Event Data
+""""""""""""""""""
+
+If this option is chosen, the output file will contain event data. The output is not an event
+workspace but rather a group workspace which contains two child event workspaces one for the
+added event data and one for the added monitor data.
```

Comment:
the above sentence is a bit clumsy. It is also missing two commas (before but and which)